spec/dradis/plugins/pentera/importer_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'spec_helper'
+
+describe Dradis::Plugins::Pentera::Importer do
```

Comment:
we're only testing the happy path, otherwise I'd suspect you'd have picked up my previous comment.

Reply:
Updated to support the failure path